README.md

```diff
@@ -164,7 +164,29 @@ public class WebServerSpec {
     }
 }
 ```
-          
+
+#### Automatic generation of CRDs
+
+To automatically generate CRD manifests from your annotated Custom Resource classes, you only need
+to add the following dependency to your project:
+
+```xml
+<dependency>
+  <groupId>io.fabric8</groupId>
+  <artifactId>crd-generator</artifactId>
+  <scope>compile</scope>
+</dependency>
+```
```

Comment:
Shouldn't we add the dependency on ```<dependency>
      <groupId>io.fabric8</groupId>
      <artifactId>kubernetes-model-common</artifactId>
    </dependency>``` here?

Reply:
Done. Can we merge this now?